cmd/atlas/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "ariga.io/atlas/cmd/atlas/cmd"
```

Comment:
merge cmd/atlas/cmd files to this package.

Reply:
Done